docs/scos/dev/tutorials-and-howtos/howtos/howto-reduce-jenkins-execution-costs-without-refactoring.md

```diff
@@ -17,17 +17,24 @@ By default, our system has a limit of two Jenkins executors for each environment
 
 Increasing the number of processes per queue can lead to issues such as Jenkins hanging, crashing, or becoming unresponsive. Although memory consumption and CPU utilization are not generally high (around 20-30%), there can be spikes in memory consumption due to a random combination of several workers simultaneously processing heavy messages for multiple stores. 
 
-There are two potential solutions to address this problem: application optimization and better background job orchestration.
+There are two potential solutions to address this problem that can be implemented simultaneously: application optimization and better background job orchestration.
 
-# Proposed Solution
+# Proposed Solution "Application optimization"
+This solution is not part of the current documentation.
+For more information please check next articles: 
+- [https://docs.spryker.com/docs/scos/dev/guidelines/performance-guidelines/architecture-performance-guidelines.html](https://docs.spryker.com/docs/scos/dev/guidelines/performance-guidelines/performance-guidelines.html)
+- [https://docs.spryker.com/docs/scos/dev/troubleshooting/troubleshooting-performance-issues/troubleshooting-performance-issues.html#prerequisites] (https://docs.spryker.com/docs/scos/dev/troubleshooting/troubleshooting-performance-issues/troubleshooting-performance-issues.html#prerequisites)
+  
+# Proposed Solution "Background job orchestration."
+## Solution descritpion
 
 ![image](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/dev/tutorials-and-howtos/howtos/howto-reduce-jenkins-execution-cost-without-refactoring/OneWorker-diagram.png)
```

Comment:
@lenadoc , @VadymSachenko can you please remove header text on the image and leave only schema? 
Thank you

Reply:
Please check spelling on the image and update